src/EditorFeatures/CSharpTest/ExtractMethod/ExtractMethodTests.cs

```diff
@@ -10920,56 +11024,25 @@ public async Task Hello()
                 {
                     var cancellationToken = CancellationToken.None;
 
-                    int i = await NewMethod(ref cancellationToken);
```

Comment:
because extract method ends up ownign the code gen here, and it just asks to synthesize a local.  which respects teh user's stated option on var-usage.  I haven't changed any of that here.  It's just how EM works.